src/core/componentCreators.js

```diff
@@ -15,27 +15,37 @@ governing permissions and limitations under the License.
 import createDataCollector from "../components/DataCollector";
 import createActivityCollector from "../components/ActivityCollector";
```

Comment:
Make sure all `OPTIONAL_COMPONENTS`, like `createActivityCollector`, have a `@skipwhen` comment that allows them to be excluded.